src/main/java/springboot/webapp/usersmanager/entities/User.java

```diff
@@ -0,0 +1,40 @@
+package springboot.webapp.usersmanager.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
```

Comment:
use one Data lombok annotation

Reply:
@Data is not recommended for JPA entities

Reply:
In this case, you have to implement Equals and Hashcode for future testing or add related lombok annotations.